index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Twitter
+        </title>
+        <script src="https://kit.fontawesome.com/34321f5008.js" crossorigin="anonymous"></script>
+        <link href="https://fonts.googleapis.com/css2?family=Oxygen:wght@700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="style.css" />
+        <script src="script.js" ></script>
+
+    </head>
+    <body>
+        <div class="content">
+            <div class="colm1">
+                <div class="icons">
+                    <a class="twitter-icon" href="#">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </div>
+            </div>
+            <div class="colm2">
+                <div class ="navbar borderStyle">Home</div>
+                <div class="newsfeed borderStyle"></div>
```

Comment:
nothing in that div! are you planning to put sth on it or what?

Reply:
yeah, I have added things to it